classes/controller/AdminController.php

```diff
@@ -2698,6 +2701,20 @@ public function setMedia($isNewTheme = false)
             ],
         ]);
 
+        $username = $this->get('prestashop.user_provider')->getUsername();
```

Comment:
I believe this part is only needed for the default theme no? since in new theme it is already handled via the twig extension. So maybe you can move this code inside the `else` block dedicated to the default theme?